### DISCUSSION TITLE
use of webpack

### DISCUSSION THREAD

**Discussion by hiro5id** [neilenns/node-deepstackai-trigger#449] [General]
Why is webpack being used for this if it is primarily a nodejs back-end application?  I am asking if there is any value in keeping it.

Webpack is running into issues while updating some dependencies.  Mainly because I've been trying to fix some glitches that I'm encountering, within in my forked version.   And also as a side effect to update insecure dependencies.

I'm wondering if Webpack is serving some purpose that I'm not understanding.  Perhaps I could remove it?

**Comment by neilenns** [comment]
I wanted to make the Docker image as small as possible so I used webpack to streamline all the node stuff down. It's not strictly necessary if you don't care about the image size.
